src/wesnoth.cpp

```diff
@@ -845,6 +845,8 @@ static int do_gameloop(const std::vector<std::string>& args)
 			break;
 		case gui2::dialogs::title_screen::REDRAW_BACKGROUND:
 			break;
+		//gui2::dialogs::title_screen::RELOAD_UI intentionally left out
```

Comment:
How is this any different from `case gui2::dialogs::title_screen::RELOAD_UI: break;`?

Reply:
Done.